desktop/core/src/desktop/lib/raz/clients_test.py

```diff
@@ -58,3 +63,39 @@ def test_check_acccess_s3_list_file(self):
 
 
   def test_check_acccess_s3_list_file_no_access(self): pass
+
+class AdlsRazClientTest(unittest.TestCase):
+
+  @classmethod
+  def setUpClass(cls):
+    if not RAZ.IS_ENABLED.get():
+      raise SkipTest
+  
+  def setUp(self):
+    self.username = 'csso_hueuser'
+  
+  def test_check_rename_operation(self):
+    with patch('desktop.lib.raz.raz_client.RazToken.get_delegation_token') as raz_token:
+      with patch('desktop.lib.raz.raz_client.requests.post') as requests_post:
+        with patch('desktop.lib.raz.raz_client.uuid.uuid4') as uuid:
+          with patch('desktop.lib.raz.raz_client.RazClient.check_access') as check_access:
+
+            reset = RAZ.API_URL.set_for_testing('https://raz_url:8000')
+            check_access.return_value = {'token': 'some_random_sas_token'}
+
+            try:
+              sas_token = AdlsRazClient(
+                username=self.username,
+                base_url='https://gethuestorage.dfs.core.windows.net'
```

Comment:
Any way to avoid introducing this base_url?
(i.e. guess from the `path` or lookup from config)

Reply:
We can guess from path (e.g. taking everything till `...dfs.windows.net`).